assembl/static2/js/app/components/svg/pointer.jsx

```diff
@@ -3,12 +3,16 @@ import React from 'react';
 class Pointer extends React.Component {
   render() {
     const position = this.props.position;
+    const marginLeft = position ? `calc(${position}% - 15px)` : 'initial';
+    const width = this.props.width ? this.props.width : '31px';
+    const height = this.props.height ? this.props.height : '46px';
+    const style = Object.assign({}, { marginLeft: marginLeft }, this.props.style);
```

Comment:
You can simplify it to :
`Object.assign({ marginLeft: marginLeft }, this.props.style)`
or even better:
`{ marginLeft: marginLeft, ...this.props.style}`